osf/models/user.py

```diff
@@ -872,6 +863,8 @@ def merge_user(self, user):
         user.merged_by = self
 
         user.save()
+        signals.user_account_merged.send(user)
+        signals.user_account_deactivated.send(self)
```

Comment:
?????

Reply:
Yeah this deserves some more comments, but since `deactivate_account` isn't called during an account merge the signal is never sent, but the user still needs to be removed from the GV side. So I have to call both signals here, for the living account to merge resources and the dead account can be disabled.

Reply:
90% sure you're deactivating the wrong user (this code looks to be giving `user`'s resources to `self`. Also...you control what GV's response to the `merge_account` message is, so you can add a step of deactivating the correct user :)

Reply:
Ug I am dumb, of course the merged account is also deactivated.

Reply:
Been there, but I would also still argue for just having the GV side mark a merged user as `deactivated` (if necessary), since there's no guarantee that any future subscriber to this signal (internal or external) will

Reply:
That makes sense merge will send one signal.